### PR TITLE
WIP: convert ITS-90 guide samples

### PR DESCRIPTION
**Pull request by manuel489**
*As requested in https://github.com/metanorma/mn-samples-bipm/issues/2*

### CODE REVIEW

**Comment by ronaldtse** [comment]
Feel free to merge when ready.

File: sources/guide-its-90/01-introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+:workgroup-acronym: CCT-TG-K
+:fullname: B Fellmuth
+:docstage: in-force
+:docsubstage: 60
+:imagesdir: images
+:mn-document-class: bipm
+:mn-output-extensions: xml,html,pdf,rxl
+:local-cache-only:
+:data-uri-image:
+
```
Comment (ronaldtse) [inline sources/guide-its-90/01-introduction.adoc:23]: Add `:si-aspect: K_k`

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: sources/guide-its-90/01-introduction.adoc (hunk 1 above)
Comment (manuel489) [reply sources/guide-its-90/01-introduction.adoc:24]: Done.
(I'm almost finished, one file remaining.)

Comment (manuel489) [reply sources/guide-its-90/01-introduction.adoc:24]: All documents are converted.

**Comment by manuel489** [comment]
All documents from https://www.bipm.org/en/committees/cc/cct/guide-its90.html are converted

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thank you @manuel489 !

**Pull request merged**